### ISSUE TITLE
Video Grid error

### ISSUE THREAD

**Issue by kuhlkrein** [GazePlay/GazePlay#1230]
**Describe the bug**
```
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - Exception in thread "JavaFX Application Thread" java.lang.UnsupportedOperationException
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at java.base/java.util.AbstractList.remove(AbstractList.java:167)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at net.gazeplay.games.videogrid.VideoGrid.launch(VideoGrid.java:104)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at net.gazeplay.ui.scenes.gamemenu.GameMenuController.chooseGame(GameMenuController.java:87)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at net.gazeplay.ui.scenes.gamemenu.GameVariantDialog.lambda$new$3(GameVariantDialog.java:145)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at com.sun.javafx.event.CompositeEventHandler$NormalEventHandlerRecord.handleBubblingEvent(CompositeEventHandler.java:218)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:80)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:238)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:191)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(CompositeEventDispatcher.java:59)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
2020-09-15 10:57:12,877 ERROR java.lang.Throwable - 	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:54)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at javafx.event.Event.fireEvent(Event.java:198)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at javafx.scene.Scene$ClickGenerator.postProcess(Scene.java:3564)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at javafx.scene.Scene$ClickGenerator.access$8200(Scene.java:3492)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at javafx.scene.Scene$MouseHandler.process(Scene.java:3860)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at javafx.scene.Scene$MouseHandler.access$1200(Scene.java:3579)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at javafx.scene.Scene.processMouseEvent(Scene.java:1849)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at javafx.scene.Scene$ScenePeerListener.mouseEvent(Scene.java:2588)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:397)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:295)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at java.base/java.security.AccessController.doPrivileged(Native Method)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$2(GlassViewEventHandler.java:434)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(QuantumToolkit.java:390)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(GlassViewEventHandler.java:433)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.glass.ui.View.handleMouseEvent(View.java:556)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.glass.ui.View.notifyMouse(View.java:942)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277)
2020-09-15 10:57:12,878 ERROR java.lang.Throwable - 	at java.base/java.lang.Thread.run(Thread.java:834)
```

**To Reproduce**
1. Create a valid directory with some video
2. Select it as video directory in gazeplay
3. Launch the video grid game

**Expected behavior**
it should play the 2 video, I've checked the error, it's due to the  `final File file = filesChooseFrom.remove(index);` I tried replacing with a get, and the get seems to work fine, maybe the files are too big to be removed from the list?

We could instead take the random index from an index list

**Comment by kuhlkrein** [comment]
ok actually the problem is here:
 final List<File> filesChooseFrom = Arrays.asList(files);
Array.asList doesn't accept the remove

**Comment by kuhlkrein** [comment]
fixed by #1231
